js-core/week-05/lesson.md

```diff
@@ -446,12 +453,14 @@ console.log(isAMentor("Rukmini")); // logs false
 
 ### `.join()`
 
-_Returns all the array values joined together in a string._
+_Returns all the array values joined together in a string. By default, this method takes no parameters and then the elements are divided with a comma `,`. If you provide it with a string parameter though, then it becomes the divider of the elements, like the example below:_
 
 ```sh
 $ node
-> ["H", "e", "l", "l", "o"].join()
-'Hello'
+> ["H", "e", "l", "l", "o"].join();
+'H,e,l,l,o'
+> ["H", "e", "l", "l", "o"].join("==");
+'H==e==l==l==o'
```

Comment:
I wonder if using `"=="` will confuse things (because it looks like to the `==` operator)? Maybe this could be changed to `"--"`?

Reply:
Ah, no, that's a fair point, I'll change to `"--"`! Thanks!

Reply:
Done in https://github.com/CodeYourFuture/syllabus/pull/264/commits/0971bedbcd51eb13c2a3dcacb05008dca3c87ff0